src/applications/vre/definitions/profileAddress.js

```diff
@@ -230,6 +230,7 @@ export const addressUiSchema = (path, checkBoxTitle, uiRequiredCallback) => {
         required: 'Please enter a valid State, Province, or Region',
       },
       'ui:options': {
+        hideEmptyValueInReview: true,
```

Comment:
added this because if the address is international, then state/province/territory is optional.